frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -204,7 +204,7 @@ ListDropdown.propTypes = {
   disabled: PropTypes.bool,
 };
 
-const useProjectOrNamespaceModel = () => {
+export const useProjectOrNamespaceModel = () => {
```

Comment:
Maybe a work in process change? At the moment this change is not needed.

```suggestion
const useProjectOrNamespaceModel = () => {
```

Reply:
This is needed to decide the use of `Namespace` or `Project` in string literals in the `NamespaceDropdown` component